community/lucene-index/src/test/java/org/neo4j/kernel/api/impl/schema/LuceneDocumentStructureTest.java

```diff
@@ -210,7 +210,7 @@ public void shouldBuildWildcardQueries() throws Exception
         WildcardQuery query = (WildcardQuery) LuceneDocumentStructure.newWildCardStringQuery( "foo" );
 
         // then
-        assertEquals( "string", query.getField() );
+        assertEquals( "0string", query.getField() );
```

Comment:
I'm still inclined on keeping the existing index structure for single-property indexes.